index.php

```diff
@@ -20,6 +21,10 @@
 <body class="selection:bg-teal-200">
 <nav class="flex justify-between items-center py-5 px-4 mb-10 border-b border-solid">
     <a href="index.php"><h1 class="text-5xl">Blog</h1></a>
+    <?php if (!isset($_SESSION['userid'])){
```

Comment:
See my other comment on Login.php about making a session class to handle stuff like this